### PR TITLE
Rearranged derma, various fixes

### PR DESCRIPTION
**Pull request by samuelWilliams99**
Looks like this now: 
image

Includes context menu populated by `CFC_FactionPanel_PopulateMenu` hook as follows:
```
hook.Add("CFC_FactionPanel_PopulateMenu", "Example", function(panel, Menu)
    
    Menu:AddSpacer()
    Menu:AddOption( "Rename" )
    Menu:AddOption( "Delete" )
    -- etc.
end)
```
image

### CODE REVIEW

**Comment by ghost** [comment]
We should get as many people's opinions on this as possible since this will be the first thing everyone sees once factions is pushed.

File: lua/cfcfactions/core/cl/cl_init.lua [hunk 1]
```diff
@@ -14,3 +14,10 @@ include( "cfcfactions/core/cl/cl_utilities.lua" )
 include( "cfcfactions/core/cl/dermas/tabs/cl_creditsderma.lua" )
 include( "cfcfactions/core/cl/dermas/tabs/cl_alertsderma.lua" )
 include( "cfcfactions/core/cl/dermas/tabs/cl_logsderma.lua" )
+include( "cfcfactions/core/cl/dermas/util/d_imagecircle.lua" )
+
+concommand.Add("cfc_factions_reload", function()
```
Comment (ghost) [inline lua/cfcfactions/core/cl/cl_init.lua:20]: What reason would we need to reload the mainview?

Comment (samuelWilliams99) [reply lua/cfcfactions/core/cl/cl_init.lua:20]: For development purposes, can be removed when we deploy, was annoying having to relog constantly as panels don't get reloaded on save

Comment (ghost) [reply lua/cfcfactions/core/cl/cl_init.lua:20]: Can you put a todo, remove on official deployment or something so I remember. Its fine to have it till then

Comment (samuelWilliams99) [reply lua/cfcfactions/core/cl/cl_init.lua:20]: Done

File: lua/cfcfactions/config/sv_config.lua [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-        sql_username = "nativeuser",
-        sql_password = "factions",
-        sql_hostname = "localhost",
-        sql_database = "cfctest",
-        sql_port = 3306
+        username = "nativeuser",
+        password = "factions",
+        hostname = "localhost",
+        database = "cfctest",
+        port = 3306
```
Comment (justinmuskopf) [inline lua/cfcfactions/config/sv_config.lua:8]: yay!

File: lua/cfcfactions/core/cl/cl_clientstartup.lua [hunk 3]
```diff
@@ -36,9 +36,17 @@ net.Receive('CFC_Fac_ToggleDerma', function(length)
...
+
 hook.Add( "PlayerButtonDown", "CFC_Fac_MenukeyDown", function( player, button )
-    if MENU_KEY == button then
+    local cTime = CurTime()
+    if MENU_KEY == button and cTime - lastHit > 0.1 then
         net.Start("CFC_Fac_RequestDerma")
         net.SendToServer()
     end
+    lastHit = cTime
 end )
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/cl/cl_clientstartup.lua:52]: Can you give me a better description of the problem?

**Comment by ghost** [comment]
I think we're going to fit less factions on, add some padding around, and change the lock icon to the one hex linked in discord.

**Comment by samuelWilliams99** [comment]
Changed UI, updated images in the pr desc

**Review by justinmuskopf — APPROVED** [review]

**Pull request merged**